var/spack/repos/builtin/packages/openresty/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Openresty(AutotoolsPackage):
+    """
+    OpenResty is a full-fledged web application server by bundling the
+    standard nginx core, lots of 3rd-party nginx modules, as well as
+    most of their external dependencies.
+    """
+
+    homepage = "https://github.com/openresty"
+    url      = "https://github.com/openresty/openresty/releases/download/v1.15.8.2/openresty-1.15.8.2.tar.gz"
+
+    version('1.15.8.2',    sha256='bf92af41d3ad22880047a8b283fc213d59c7c1b83f8dae82e50d14b64d73ac38')
+    version('1.15.8.1rc1', sha256='9686805d46131dee33a279e517f995decebde7753ad624b1e18ad4be9657ecc1')
```

Comment:
Probably don't need this release candidate now that the stable version has been released.